r2/r2/templates/printable.html

```diff
@@ -101,6 +101,9 @@
       rowclass += " deleted"
     if hasattr(what, "saved") and what.saved:
       rowclass += " saved"
+    if c.user.gold and hasattr(what, "saved") and what.saved and hasattr(what, "saved_category"):
+      item_saved_category = "no-category" if not what.saved_category else what.saved_category
```

Comment:
It's impossible for category names to have a "-" in them, so this isn't really an issue. And it seemed nicer than " saved-" with no category suffix.